src/site/content/en/blog/multi-screen-window-placement/index.md

```diff
@@ -0,0 +1,433 @@
+returns a [`Screen`](https://developer.mozilla.org/en-US/docs/Web/API/Screen) object. This is the
+output on my MacBook Air 13″:
+
+```js
+window.screen;
+/* Output from my MacBook Air 13″:
+  availHeight: 975
+  availLeft: 0
+  availTop: 23
+  availWidth: 1680
+  colorDepth: 30
+  height: 1050
+  id: ""
+  internal: false
+  left: 0
+  orientation: ScreenOrientation {angle: 0, type: "landscape-primary", onchange: null}
+  pixelDepth: 30
+  primary: false
+  scaleFactor: 2
+  top: 0
+  touchSupport: false
+  width: 1680
+*/
+```
+
+Like most people working in tech, I have had to arrange myself with the new work reality and install
+my personal home office. Mine looks like on the photo below (if you are interested, you can read the
+[full details about my setup](https://blog.tomayac.com/2020/03/23/my-working-from-home-setup-during-covid-19/)).
+The iPad next to my MacBook Air is connected to the laptop via
+[Sidecar](https://support.apple.com/en-us/HT210380), so whenever I need to, I can quickly turn the
+iPad into a second screen.
+
+<figure class="w-figure">
+  <img src="./desk.jpg" alt="School bench on two chairs. On top of the school bench are shoe boxes with a laptop on top and two iPads surrounding it.">
+  <figcaption class="w-figcaption">A multi-screen setup.</figcaption>
+</figure>
+
+If I want to place the popup window from the
+[code sample](/multi-screen-window-placement/#the-problem)
+above onto the second screen to now actually make actual use of the bigger screen real estate, I can do
+so as follows:
+
+```js
+popup.moveTo(2500, 50);
+```
+
+At this stage, this is a rough guess, since there is no way to know the dimensions of the second
+screen. The info from `window.screen` purely covers the built-in screen, but not the iPad screen
+attached via Sidecar. The reported `width` of the built-in screen was `1680` pixels, so moving to
+`2500` pixels *might* work to shift the window over to the iPad, since _I_ happen to know that it is
+located on the right of my MacBook Air. How to do this in the general case? Turns out, there is a
+better way than guessing, which is paved by the Multi-Screen Window Placement API.
+
+### Feature detection
+
+To check if the Multi-Screen Window Placement API is supported, use:
+
+```js
+if ("getScreens" in window) {
+  // The Multi-Screen Window Placement API is supported.
+}
+```
+
+### The `window-placement` permission
+
+Before I can use the Multi-Screen Window Placement API, I must ask the user for permission to do so.
+The new `window-placement` permission can be queried with the
+[Permissions API](https://developer.mozilla.org/en-US/docs/Web/API/Permissions_API) like so:
+
+```js
+let granted = false;
+try {
+  const { state } = await navigator.permissions.query({ name: "window-placement" });
+  granted = state === "granted";
+} catch {
+  // Nothing.
+}
+```
+
+The browser can choose to show the permission prompt dynamically upon the first attempt to use any
```

Comment:
It is not clear in the rest of the article when user is prompted. I assume it us when `getScreens` is called, but maybe also `isMultiScreen`.

Reply:
Still [up for debate](https://webscreens.github.io/window-placement/#issue-c601b517:~:text=Permission%20check%20for%20above%3F,-partial).